src/presentation/ui/component/form/selectInput.templ

```diff
@@ -0,0 +1,136 @@
+package componentForm
+
+templ SelectInput(
+	id, label, bindValuePath string,
+	options []string,
+	includeBlankOption, denseMode bool,
```

Comment:
When analyzing the code to make this modification, I noticed that we have a use case where adding an empty option would be a bad idea. In the advanced mapping creation options, you can configure the **MatchPattern**. Since these are advanced settings within an expandable div that is initially closed, this option becomes almost "invisible" to the typical user, and is only necessary for more technical users. In this case, it doesn't make sense to add an empty option and force the user to select it, given that it is an advanced and therefore optional setting.

Reply:
Now all selects have a blank option.